src/app/pages/svangerskapspenger/hvemKanFa/HvemKanFåSvangerskapspenger.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { InjectedIntlProps, injectIntl } from 'react-intl';
+import PanelMedIllustrasjon from '../../../components/panel-med-illustrasjon/PanelMedIllustrasjon';
+import SvgMask from '../../../components/svg-mask/SvgMask';
+import BEMHelper from '../../../utils/bem';
+import Innhold, { getSource } from '../../../utils/innhold/Innhold';
+import getTranslation from '../../../utils/i18nUtils';
+import CustomSVG from '../../../utils/CustomSVG';
+import './hvemKanFåSvangerskapspenger.less';
+
+const cls = BEMHelper('hvemkanfaSvangerskapspenger');
+const seksjonsbilde = require('../../../assets/ark/ark-arbeider.svg').default;
+const checkmarkIcon = require('../../../assets/icons/checkmark.svg').default;
+
+interface Props {
+    id: string;
+}
+
+const HvemKanFåSvangerskapspenger: React.StatelessComponent<Props & InjectedIntlProps> = ({
+    id,
+    intl
+}) => {
+    return (
+        <PanelMedIllustrasjon
+            id={id}
+            title={getTranslation(
+                'om_svangerskapspenger.hvem_kan_fa_svangerskapspenger.tittel',
+                intl
+            )}
+            svg={<SvgMask svg={seksjonsbilde} anchorToBottom={true} />}>
+            <div className={cls.element('alignLeft')}>
+                <Innhold
+                    source={getSource(
+                        'svangerskapspenger/hvem-kan-fa-svangerskapspenger/ingress',
+                        intl
+                    )}
+                />
+            </div>
+            <div className={cls.element('kravTilSvangerskapspenger')}>
+                {kravTilSvangerskapspengerStringPath.map((krav) => (
+                    <div key={krav} className={cls.element('krav')}>
+                        <KravTilSvangerskapspenger ingress={getSource(krav, intl)} />
```

Comment:
Må du wrappe `KravTilSvangerskapspenger` i en div? Det gjør at du anvender `className={cls.element('krav')}` to ganger, er dette meningen?